docs/source/release_notes/release-notes.rst

```diff
@@ -41,39 +41,53 @@ develop
+           See :ref:`oracle_table_mapping` for details on how table names are mapped.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1076>`__)
+
     *    - |improved|
          - Added a significant amount of logging aimed at tracking down the ``MultipleRunningTimestampServicesError``.
            If clients are hitting this error, then they should add trace logging for ``com.palantir.timestamp``.
            These logs can also be directed to a separate file, see the :ref:`documentation <logging-configuration>` for more details.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1098>`__)
 
-    *    - |deprecated|
-         - ``TableReference.createUnsafe`` is now deprecated. ``createWithEmptyNamespace`` or ``createFromFullyQualifiedName`` should be used instead.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1121>`__)
-           
-
     *    - |improved|
          - Random redirection of queries when retrying a Cassandra operation now retries said queries on distinct
            hosts. Previously, this would independently select hosts randomly, meaning that we might unintentionally
            try the same operation on the same server(s).
-
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1139>`__)
-    *    - |new|
-         - Added the ability to do custom performance logging for Sweep. This is currently only available for
-           applications that call the ``BackgroundSweeperImpl`` constructor directly. Otherwise a
-           ``NoOpBackgroundSweeperPerformanceLogger`` is used.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1054>`__)
 
-    *    - |new|
-         - The KVS migration command can now be run as an offline cli using the ``--offline`` flag.
+    *    - |improved|
+         - AtlasDB clients can start even if one Cassandra node is unreachable.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1045>`__).
 
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1149>`__)
+    *    - |improved|
+         - Actions performed by the ``Scrubber`` are now logged at debug instead of info.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1137>`__)
 
-    *   - |improved|
-        - We now support Oracle for products with all valid schemas. Oracle table names exceeding 30 characters are now
-          mapped to shorter names by truncating and appending a sequence number.
-          See :ref:`oracle_table_mapping` for details on how table names are mapped.
-          (`Pull Request <https://github.com/palantir/atlasdb/pull/1076>`__)
+    *    - |improved|
+         - Update teh Cassandra partitioner for compatibility with newer versions of Cassandra.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1122>`__)
+
+    *    - |fixed|
+         - Fixed an issue where leader election threads were not correctly marked as daemon threads.
```

Comment:
I don't personally think this is impactful enough to go in the release notes, although you might want to follow up with @tjwilson90 as he made the PR.